third_party/skymarshal/cmake/parse_types.py

```diff
@@ -1,5 +1,7 @@
 import argh
-
+import sys
+import os
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
Hmm.  This should be handled by the PYTHONPATH here in skymarshal.cmake:

```cmake
  # NOTE(aaron):  Who knows if this'll work if there are spaces in the PYTHONPATH?  I don't
  set(SKYMARSHAL_PYTHON env PYTHONPATH=$ENV{PYTHONPATH}:${CMAKE_CURRENT_FUNCTION_LIST_DIR}/../.. ${SYMFORCE_PYTHON})

  execute_process(
    COMMAND ${SKYMARSHAL_PYTHON} ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/parse_types.py ${lcmtypes_dir}
    OUTPUT_VARIABLE TYPES_TO_GENERATE
    COMMAND_ERROR_IS_FATAL ANY
  )
  string(REPLACE "\n" ";" TYPES_TO_GENERATE_LIST ${TYPES_TO_GENERATE})
```

And it seems like CMAKE_CURRENT_FUNCTION_LIST_DIR at least hasn't changed in newer cmake

Reply:
Probably because of 

![](https://imgs.xkcd.com/comics/python_environment_2x.png)

I'll go ahead and assume it's an issue on my end and remove that change